npc/quests/quests_moscovia.txt

```diff
@@ -7524,6 +7592,8 @@ moscovia,178,127,0	script	The Blacksmith#rus06	1_M_SMITH,{
 			mes "And don't forget to wear these!";
 			delitem Steel,25;
 			rhea_rus_main = 8;
+			if (BaseClass != Job_Thief)
```

Comment:
i think the script is simply outdated here. I found a lot more not fitting official and i think this might is related to the different items required for the different classes.